src/Expanders.jl

```diff
@@ -530,14 +530,24 @@ end
 # --------
 
 function Selectors.runner(::Type{ExampleBlocks}, x, page, doc)
+    matched = match(r"^@example(?:\s+([^\s;]+))?\s*(;.*)?$", x.language)
+    matched === nothing && error("invalid '@example' syntax: $(x.language)")
+    name, kwargs = matched.captures
     # The sandboxed module -- either a new one or a cached one from this page.
-    name = match(r"^@example[ ]?(.*)$", first(split(x.language, ';', limit = 2)))[1]
     mod = Utilities.get_sandbox_module!(page.globals.meta, "atexample", name)
     sym = nameof(mod)
     lines = Utilities.find_block_in_file(x.code, page.source)
 
-    # "parse" keyword arguments to example (we only need to look for continued = true)
-    continued = occursin(r"continued\s*=\s*true", x.language)
+    # "parse" keyword arguments to example
+    continued = false
+    ansicolor = get(stdout, :color, false)::Bool
```

Comment:
I wonder if this is a useful check, does `stdout` on e.g. CI always give `true` here? Perhaps  `true` can just be default and if you don't want it you explicitly say that with `ansicolor=false`?

Reply:
Actually, lets check `Base.JLOptions().color`, and if it is `2` then disable (e.g. if you started with `--color=no`)

Reply:
```suggestion
    ansicolor = Base.JLOptions().color != 2 # disable if --color=no
```